.github/workflows/checks.yml

```
@@ -10,7 +10,7 @@ permissions: read-all
 jobs:
   audit:
     name: Audit
-    uses: ericcornelissen/shescape/.github/workflows/reusable-audit.yml@main
+    uses: ericcornelissen/shescape/.github/workflows/reusable-audit.yml@audit-ossf-scorecard
```

This change **must** be reverted before this Pull Request is merged.

```suggestion
    uses: ericcornelissen/shescape/.github/workflows/reusable-audit.yml@main
```

---

Invalidated by 4bee8e5fae4567d860b9be96ea62181ebcc174e1